checker/tests/run-pass/assume_provably_false.rs

```diff
@@ -0,0 +1,5 @@
+pub fn main() {
+    let a = 5;
+    assume!(a < 5); //this assumption is provably false and it will be ignored
```

Comment:
To make the test case real, it should be `//~ this assumption is provably false and it will be ignored`

It also seems that no message is generated, because the test case passes and because the expected error should be `assumption is provably false and it will be ignored in the assertion` according to the code.

Reply:
I misunderstood how tests are executed. Thanks for pointing this out.
Now, I see how it works. 
Fixed it.